mypy/checker.py

```diff
@@ -1894,6 +1892,19 @@ def check_init_subclass(self, defn: ClassDef) -> None:
             # all other bases have already been checked.
             break
 
+    def check_enum(self, defn: ClassDef) -> None:
+        assert defn.info.is_enum
+        if defn.info.fullname not in ENUM_BASES:
+            for sym in defn.info.names.values():
+                if (isinstance(sym.node, Var) and sym.node.has_explicit_value and
+                        sym.node.name == '__members__'):
```

Comment:
Let's add a comment: why don't we allow `__members__`

Reply:
So, to clarify:

```python
>>> import enum
>>> class E(enum.Enum):
...   __members__ = (1, 2)
... 
>>> class D(E):
...   __members__ = (3, 4)
... 
>>> D.__members__
mappingproxy({})
```

It is not a runtime error. But, it does not make sense, right?

Reply:
Yeah, exactly. `__members__` will always be overwritten. So it doesn't make sense to try to declare any value to it.